src/App.js

```diff
@@ -1,13 +1,43 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import PhotoCard from "./components/PhotoCard";
 import "./App.css";
+import logo from "./img/NASA_logo.png";
+import {config} from "./config/config";
 
 function App() {
+  const [data, setData] = useState(null);
+  const [date, setDate] = useState('');
+
+  
+
+  useEffect(() => {
+    var mykey = config.MY_KEY;
+    axios
```

Comment:
This is a great practice. Eventually we'll set it up to read these from a .env file later in class